guides/Predictive-analytics-with deepnote.md

```diff
@@ -0,0 +1,51 @@
+---
+description: Deepnote is a versatile notebook that revolutionizes analytics and data science workflow by seamlessly combining the power of SQL and Python with a wide range of data frameworks. 
+---
+
+# Predictive analytics from application Telementry
+
+## Scope
+The focus of this guide is to display the ease of integrating Hydra to [Deepnote](https://deepnote.com/docs) to perform analytics and predictions using python scripts.
+
+In order to make predictions a simple salary dataset was chosen with two columns 'Years of experience' and 'Salary'.
+
+## Setup
+
+- Ensure you have the details of your hydra instance handy.
+- Sign up for your free instance at [Deepnote](https://deepnote.com/sign-up)
+
+### Building the table with Hydra
+
+1. A sample data from [Kaggle](https://www.kaggle.com/datasets/rsadiq/salary) would be used for this guide.Download the csv file.
+
+2. Connect to your Hydra database and import the CSV file to the database using the steps for importing a CSV file outlined in the [Documentation](https://docs.hydra.so/centralize-data/load/from-local-csv-file)
+
+### Creating the Connection
+
+Now we can connect the Postgres Database to Deepnote
+
+1. Start by [Signing Up](https://deepnote.com/sign-up) and using the 14 days trial or Logging in if you already have an account.
```

Comment:
Create your 14 day free trial of Deepnote or sign into your existing Deepnote account.